crates/transaction-pool/src/pool/txpool.rs

```diff
@@ -1442,8 +1443,8 @@ impl<T: PoolTransaction> AllTransactions<T> {
         on_chain_nonce: u64,
     ) -> InsertResult<T> {
         assert!(on_chain_nonce <= transaction.nonce(), "Invalid transaction");
-
-        let mut transaction = self.ensure_valid(transaction)?;
+        let pool_config = PoolConfig::default();
+        let mut transaction = self.ensure_valid(transaction, pool_config)?;
```

Comment:
hmm, this looks odd, we shouldn't need to pass a new config on every insert

Reply:
> hmm, this looks odd, we shouldn't need to pass a new config on every insert

thanks, solved !